### DISCUSSION TITLE
Welcome to Literate-Programming Discussions!

### DISCUSSION THREAD

**Discussion by alapo** [alapo/Literate-Programming#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hganshorn2** [comment]
Hi, I'm Heather, and I'm a librarian at the University of Calgary. I'm an R newbie, but taking this course to hopefully build some data skills.

**Reply by alapo** [reply]
That's amazing Heather! I will be very interested in getting your feedback as we progress through the workshop.

**Reply by alapo** [reply]
Hey Heather here are two packages I found while browsing that may be of interest to you

1. [revtools](https://github.com/mjwestgate/revtools),  [see published Article](https://onlinelibrary.wiley.com/doi/abs/10.1002/jrsm.1374)
2. [metagear](https://github.com/mjlajeunesse/metagear),  [On YouTube](https://www.youtube.com/channel/UCvoAaXbImj95zwgdguxbrPA)
3. [Another post](http://meta-evidence.co.uk/systematic-review-meta-analysis-and-r/)

**Comment by hganshorn2** [comment]
Thanks! Will definitely check these out. I’m hoping desperately that we’ll be able to keep our institutional licence to Covidence in the face of the budget cuts that are no doubt coming, but good to know that there are some R tools that provide another option.

Heather
